client/src/main.rs

```diff
@@ -106,7 +106,7 @@ async fn main() -> Result<(), anyhow::Error> {
                         token_contract: MULTICHAIN_TOKEN,
                         timeout: 3 * 60 * 60,
                     })
-                    .gas(100_000)
+                    .gas(1_000_000)
```

Comment:
Why 1m gas?

Reply:
I got an out of gas error when I ran it with the 100k gas, I'll revert it back and try again, just want to test the deployments first

Reply:
This was the tx https://sepolia.etherscan.io/tx/0x4c4bf173237d31354e4477055060eebfbd91e8793d86dce88d6410e630faf527

Reply:
I'll set it to 150k gas, that seems to work.